python/dolfinx/graph.py

```diff
@@ -28,10 +31,34 @@
     pass
 
 
-__all__ = ["adjacencylist", "partitioner"]
+__all__ = ["AdjacencyList", "adjacencylist", "partitioner"]
+
+
+class AdjacencyList:
+    _cpp_object: Union[_cpp.la.AdjacencyList_int32, _cpp.la.AdjacencyList_int64]
```

Comment:
Would be great if we could add some docstrings to this class (as that is the main motivation for having a Python wrapper).